### PR TITLE
chore: create oryx updates

### PR DESCRIPTION
**Pull request by dunqan**
Updates `create-oryx` utility:
- use new url for downloading template
- check for latest version and warn
- always download latest template

Closes [HRZ-90610](https://spryker.atlassian.net/browse/HRZ-90610)

[HRZ-90610]: https://spryker.atlassian.net/browse/HRZ-90610?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/template/create-oryx/package.json [hunk 1]
```diff
@@ -1,13 +1,14 @@
 {
   "name": "create-oryx",
-  "version": "0.21.0",
+  "version": "0.33.0",
```
Comment (tolerants) [inline libs/template/create-oryx/package.json:3]: Wow... 12 versions. Maybe it make sense to use rc for testing? :)

File: libs/template/create-oryx/src/check-version.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import latestVersion from 'latest-version';
```
Comment (tolerants) [inline libs/template/create-oryx/src/check-version.ts]: Do we really need a dependency for it?

File: libs/template/create-oryx/package.json [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
   "type": "module",
   "author": "Spryker Systems GmbH",
   "license": "See license in LICENSE",
   "description": "Create Spryker Oryx Apps",
   "bin": "./bin/main.cjs",
   "devDependencies": {
-    "@spryker-oryx/cli": "*"
+    "@spryker-oryx/cli": "*",
```
Comment (tolerants) [inline libs/template/create-oryx/package.json]: Why not `latest`?

**Review by tolerants — CHANGES_REQUESTED** [review]

File: libs/template/create-oryx/package.json (hunk 1 above)
Comment (dunqan) [reply libs/template/create-oryx/package.json:3]: Not 12, as new one started with `0.30`.
It was not about testing, as testing was done with verdaccio, but troubleshoting deployment issues.
Not sure if there is a point for rc either, as nothing/noone is relying on this version.

File: libs/template/create-oryx/package.json (hunk 3 above)
Comment (dunqan) [reply libs/template/create-oryx/package.json]: It doesn't make a big difference, as we build it internally anyway. But techically "*" is more correct, as we don't use latest stable release, but what we have in the monorepo at the time of the build.

File: libs/template/create-oryx/src/check-version.ts (hunk 2 above)
Comment (dunqan) [reply libs/template/create-oryx/src/check-version.ts]: Define "really". The really,  "really" we do not :)
It's only a dev-dependency, it's not affecting anything else.

**Review by tolerants — APPROVED** [review]

**Pull request merged**